apps/examples/testcase/le_tc/kernel/tc_libc_pthread.c

```diff
@@ -766,11 +766,14 @@ static void tc_libc_pthread_pthread_mutexattr_getprotocol(void)
 	TC_ASSERT_EQ("pthread_mutexattr_init", ret_chk, OK);
 
 #ifdef CONFIG_PRIORITY_INHERITANCE
+	attr.proto = PTHREAD_PRIO_INHERIT;
 	ret_chk = pthread_mutexattr_getprotocol(&attr, &mutexattr_protocol);
-	TC_ASSERT_EQ("pthread_mutexattr_getprotocol", ret_chk, PTHREAD_PRIO_INHERIT);
+	TC_ASSERT_EQ("pthread_mutexattr_getprotocol", ret_chk, OK);
```

Comment:
These shall get and set the protocol attribute of a mutex attributes object pointed to by attr which was previously created by the function pthread_mutexattr_init().
Upon successful completion, the pthread_mutexattr_getprotocol() and pthread_mutexattr_setprotocol() functions shall return zero; otherwise, an error number shall be returned to indicate the error.
Reference LLink:
http://pubs.opengroup.org/onlinepubs/009695399/functions/pthread_mutexattr_getprotocol.html